include/random_walks/gaussian_accelerated_billiard_walk.hpp

```diff
@@ -67,9 +68,15 @@ struct GaussianAcceleratedBilliardWalk
     struct Walk
     {
         typedef typename Polytope::PointType Point;
-        typedef typename Polytope::DenseMT DenseMT;
+        typedef typename Polytope::MT MT;
+        typedef typename Eigen::Matrix<double, Eigen::Dynamic, Eigen::Dynamic> DenseMT;
         typedef typename Polytope::VT VT;
         typedef typename Point::FT NT;
+        using AA_type = std::conditional_t< std::is_same_v<MT, typename Eigen::SparseMatrix<NT, Eigen::RowMajor>>, typename Eigen::SparseMatrix<NT>, DenseMT >; 
+        // AA is sparse colMajor if MT is sparse rowMajor, and Dense otherwise
+        using AE_type = std::conditional_t< std::is_same_v<MT, typename Eigen::SparseMatrix<NT, Eigen::RowMajor>> && std::is_base_of<Eigen::SparseMatrixBase<E_type>, E_type>::value, typename Eigen::SparseMatrix<NT, Eigen::RowMajor>, DenseMT >; 
+      //                                  (               (                                (                   ))                   (                       (      )        )                                     (                   )          )
```

Comment:
something is missing in the comment? Or is it some LISP code ;) ?

Reply:
aaah, sorry about that :)), I'll remove it in the next commit soon, I just had some issues with the opening and closing of <> brackets since the line is so long, so ! wrote that comment below to use the syntax highlight to see which bracket pairs up with which other one so I could find out where I was missing a bracket.